### PR TITLE
Add Random::Formatter#phrase

### PR DESCRIPTION
**Pull request by nobu**
The method generates a random pass phrase.

### CODE REVIEW

File: lib/random/formatter.rb [hunk 1]
```diff
@@ -212,4 +218,74 @@ def alphanumeric(n=nil)
...
+  # random phrase string to be generated.
+  #
+  # If _n_ is not specified or is nil, 40 is assumed.  If _n_ is a
+  # Range, the length will be chosen in that range randomly.  The
+  # default length may be larger in the future.
+  #
+  # The result will consist of chunks upto _chunk_ bytes separated by
+  # the elements of _separators_.  The chunks consist of alpha-numeric
+  # characters, except for the elements of _exclude_ unless it is
+  # falsy.  The last chunk may longer than _chunk_.  _separators_
```
Comment (olleolleolle) [inline lib/random/formatter.rb:237]:
```suggestion
  # falsy.  The last chunk may be longer than _chunk_.  _separators_
```

File: lib/random/formatter.rb [hunk 2]
```diff
@@ -370,4 +376,80 @@ def alphanumeric(n = nil, chars: ALPHANUMERIC)
...
   end
+
+  # Default chunk size for #phrase
+  CHUNK_SIZE = 5
+
+  # Default punctuation joining chunks for #phrase
+  PUNCT = %w[! - + / . ^ ~].freeze
+
+  # Default excluded characters for #phrase
+  EXCLUDE = %w"0 O o 1 l".freeze
```
Comment (olleolleolle) [inline lib/random/formatter.rb:387]: Minor: Is there a specific style reason to have another %w list delimiter for EXCLUDE than what PUNCT is using?

Comment (nobu) [reply lib/random/formatter.rb:387]: Of course nothing ;)

**Comment by nobu** [comment]
It may be nice if this method accepts `chars:`, like as `Random::Formatter#alphanumeric`.

**Pull request closed without merging**